dockerfiles/che/Dockerfile

```diff
@@ -9,7 +9,7 @@
 #   Codenvy S.A
 #
 # To build it, run in the repository root:
-#  `docker build -t codenvy/che-server .`
+#  `docker build -t eclipse/che-server .`
```

Comment:
the issue #2977 is to not build anymore the docker images from the root but from each folder
the build.sh script is handling that. Maybe I should drop the build instruction from the dockerfile by using docker build but only using build.sh